src/NServiceBus.TransactionalSession.AcceptanceTests/TransactionSessionDefaultServer.cs

```diff
@@ -15,7 +16,10 @@ public class TransactionSessionDefaultServer : DefaultServer
             Func<EndpointConfiguration, Task> configurationBuilderCustomization) =>
             base.GetConfiguration(runDescriptor, endpointConfiguration, async configuration =>
             {
-                configuration.EnableTransactionalSession();
+                // Explicitly enforcing the type to be scanned. Otherwise the scanner would not pick it up in the acceptance tests
```

Comment:
@timbussmann I could also make the feature internal but my knowledge with the feature discovery is a bit rusty. I had to force here both things into place; otherwise they won't be taken up by the scanning of the ATTs. That also makes me suspicious of what would happen if we would make the type internal once we try to run ATTs in the downstream persistence repositories. Maybe you have some ideas here?

Reply:
to my understanding `TransactionalSession` shouldn't require to be included explicitly, the default behavior is to exclude all types from the acceptance test repo from scanning (except the current test file) so other features shouldn't be affected.